### PR TITLE
Avoid unnecessary boxing in GlyphRun.cs

### PR DESCRIPTION
**Pull request by lindexi**
## Regression

No

## Testing

Just CI.

## Risk

Low.

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs [hunk 1]
```diff
@@ -1269,9 +1269,9 @@ public Rect ComputeInkBoundingBox()
 
             if ((_flags & GlyphRunFlags.CacheInkBounds) != 0)
             {
-                if (_inkBoundingBox != null)
+                if (_inkBoundingBox.HasValue)
                 {
-                    return (Rect)_inkBoundingBox;
+                    return _inkBoundingBox.Value;
```
Comment (gfoidl) [inline src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]:
```suggestion
                    return _inkBoundingBox.GetValueOrDefault();
```
`Value` repeats the check for `HasValue` which is already done.

Comment (ThomasGoulet73) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]: @gfoidl Isn't this supposed to be optimized by the JIT in .Net 6 by removing redundant checks when inlining ? Look for "GetValueOrDefault" in the [.Net 6 performance improvements blog post](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-6/)

Comment (gfoidl) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]: Oh, you are right, the JIT will do this now for us. Thanks for the reminder! (I'm too used to that pattern with GetValueOrDefault...)

Comment (lindexi) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]: @gfoidl It means I need not do any change?

Comment (gfoidl) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]: @lindexi yeah, leave it as is (it's fine so).
Sorry for the confusion.

Comment (lindexi) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphRun.cs:1274]: Thank you @gfoidl

**Comment by lindexi** [comment]
Thank you @vishalmsft

**Pull request closed without merging**